src/components/views/dialogs/ChangelogDialog.js

```diff
@@ -58,13 +62,20 @@ export default class ChangelogDialog extends React.Component {
         const QuestionDialog = sdk.getComponent('dialogs.QuestionDialog');
```

Comment:
Something happened to the indentation in this function. While you're here, mind going through and fixing it?

Reply:
Hmm, at first I suspected tabs or something... But it looks correct in my editor and even when viewing the [raw file](https://github.com/matrix-org/matrix-react-sdk/blob/f2019c44fde3075eb69999fa3daaf283c88ced15/src/components/views/dialogs/ChangelogDialog.js), so maybe a GH issue in the review UI...? 😰

Reply:
Looks like it :(

/me goes to complain to github